kyuubi-server/src/main/scala/org/apache/kyuubi/server/ui/JettyServer.scala

```diff
@@ -78,7 +78,8 @@ object JettyServer {
       host: String,
       port: Int,
       poolSize: Int,
-      stopTimeout: Long): JettyServer = {
+      stopTimeout: Long,
+      shouldSendJettyVersion: Boolean = true): JettyServer = {
```

Comment:
shouldSendJettyVersion => sendServerVersion

do we really need such a default value?

Reply:
Hi @pan3793 ，

Thanks for your suggestion. As long as there are only 2 places calling the constructor of JettyServer. We could remove the default value for the new introduced sendServerVersion parameter.